projects/storefrontapp-e2e-cypress/cypress/integration/mobile/my-account/order-history-no-orders-flow-mobile.e2e-spec.ts

```diff
@@ -0,0 +1,47 @@
+import { formats } from '../../../sample-data/viewports';
+import { user } from '../../../sample-data/checkout-flow';
+import { register, login } from '../../../helpers/auth-forms';
+import { checkBanner } from '../../../helpers/homepage';
+
+describe.only(`${formats.mobile.width +
+  1}p resolution - Order History with no orders`, () => {
+  const loginLink = 'cx-login [role="link"]';
+  const orderHistoryLink = '/my-account/orders';
+
+  before(() => {
+    cy.window().then(win => win.sessionStorage.clear());
+    cy.visit('/');
+
+    cy.get(loginLink).click();
+    cy.get('cx-page-layout')
+      .getByText('Register')
+      .click();
+    register(user);
+
+    cy.selectUserMenuOption('Sign Out');
+  });
+
+  beforeEach(() => {
+    cy.viewport(formats.mobile.width, formats.mobile.height);
+  });
+
+  it('should redirect to login page if user is not logged in', () => {
```

Comment:
Extract from this new mobile tests commands identical with regression test to helper and invoke them instead of this duplication.